src/Business Foundation/App/NoSeries/src/Setup/NoSeries.Page.al

```diff
@@ -0,0 +1,302 @@
+﻿// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+page 456 "No. Series"
+{
+    AdditionalSearchTerms = 'numbering,number series';
+    ApplicationArea = Basic, Suite;
+    Caption = 'No. Series';
+    PageType = List;
+    RefreshOnActivate = true;
+    SourceTable = "No. Series";
+    UsageCategory = Administration;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(Control1)
+            {
+                ShowCaption = false;
+                field("Code"; Rec.Code)
+                {
+                    ApplicationArea = Basic, Suite;
```

Comment:
Is ApplicationArea on page field level still needed when specified on page level?